### PR TITLE
Implement edit details

### LINKED ISSUE #300: Render edit details
https://github.com/cfpb/hmda-platform/issues/717 will provide details about each edit:
- [x] desc (#301)
- [ ] fields
- [ ] values

We'll need to modify the rendering of edits to include this information.

### LINKED ISSUE #344: Group by row failing
When using the "Group by row" button nothing renders and 2 errors show in the console:

The first is:
```
Warning: Failed prop type: Invalid prop `cell` supplied to `EditsTableCell`.
    in EditsTableCell (created by EditsTableRow)
    in EditsTableRow (created by EditsTable)
    in tbody (created by EditsTable)
    in table (created by EditsTable)
    in div (created by EditsTable)
    in EditsTable (created by EditsTableWrapper)
    in div (created by EditsTableWrapper)
    in div (created by EditsTableWrapper)
    in div (created by EditsTableWrapper)
    in EditsTableWrapper (created by t)
    in div (created by t)
    in t (created by Connect(t))
    in Connect(t) (created by t)
    in div (created by t)
    in div (created by t)
    in div (created by t)
    in div (created by t)
    in t (created by Connect(t))
    in Connect(t) (created by RouterContext)
    in div (created by t)
    in t (created by Connect(t))
    in Connect(t) (created by RouterContext)
    in RouterContext (created by Router)
    in Router
    in r
    in t
```

The next is:
```
Error: Objects are not valid as a React child (found: object with keys {Agency Code}). If you meant to render a collection of children, use an array instead or wrap the object using createFragment(object) from the React add-ons. Check the render method of `EditsTableCell`.
    at invariant (invariant.js:38)
    at traverseAllChildrenImpl (traverseAllChildren.js:144)
    at traverseAllChildren (traverseAllChildren.js:172)
    at Object.instantiateChildren (ReactChildReconciler.js:70)
    at ReactDOMComponent._reconcilerInstantiateChildren (ReactMultiChild.js:187)
    at ReactDOMComponent.mountChildren (ReactMultiChild.js:226)
    at ReactDOMComponent._createInitialChildren (ReactDOMComponent.js:691)
    at ReactDOMComponent.mountComponent (ReactDOMComponent.js:516)
    at Object.mountComponent (ReactReconciler.js:46)
    at ReactCompositeComponentWrapper.performInitialMount (ReactCompositeComponent.js:371)
(anonymous) @ index.js:630
```

### PR DESCRIPTION
**Pull request by wpears**
Uses the placeholder data responses from the backend to render more complete tables.

Closes #300 

Closes #344

### CODE REVIEW

**Comment by awolfe76** [comment]
This is really nice!

I noticed something though ... easier with a screenshot 

edits-by-row

**Comment by awolfe76** [comment]
A couple of other things:
- Do we need the edit descriptions like we have in the "group by edit"?
- The column widths vary a lot, we should try to set some widths where we can.
- The main heading ("4 Edits by Row") sounds weird. It should be something like "4 Rows have Edits".
- The headings for the rows ("2 9471480396 edits found.") sound weird too. These should be something like "rowId has 2 edits." or "2 edits found in rowId."
- The csv download for edits by row isn't a valid endpoint. Not exactly sure what we should do in this case ... may require updates on the back-end.

These can definitely handled by other issues/PRs, just wanted to mention them. I think [this one](https://github.com/cfpb/hmda-platform-ui/pull/334#issuecomment-274905727) needs fixed though.

File: src/js/components/EditsTable.jsx [hunk 1]
```diff
@@ -38,29 +39,23 @@ const renderHeader = (props) => {
   return <tr>{cells}</tr>
 }
 
-const renderBody = (props) => {
-  if (props.type === 'macro') {
-    return props.edits.map((macro, i) => {
+export const renderBody = (edits, type) => {
```
Comment (awolfe76) [inline src/js/components/EditsTable.jsx:42]: Not sure I follow why the `export` here (and others)?

File: src/js/components/EditsTable.jsx [hunk 2]
```diff
@@ -2,29 +2,30 @@ import React, { PropTypes } from 'react'
 
 import EditsTableRow from './EditsTableRow.jsx'
 
-const formatHeader = (text) => {
+export const formatHeader = (text) => {
```
Comment (awolfe76) [inline src/js/components/EditsTable.jsx:5]: Is this being used?

**Comment by awolfe76** [comment]
Also seeing "undefined undefined edits found." for the macro edits subheading. For macro is shouldn't be there.

macro-issue

File: src/js/components/EditsTable.jsx (hunk 1 above)
Comment (wpears) [reply src/js/components/EditsTable.jsx:42]: For unit tests

**Review by awolfe76 — APPROVED** [review]
👍

**Pull request merged**